src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -255,7 +255,7 @@ public static void createTempSourceBQTable() throws IOException, InterruptedExce
       records.append(" (").append(index).append(", ").append((int) (Math.random() * 1000 + 1)).append(", '")
         .append(UUID.randomUUID()).append("'), ");
     }
-    BigQueryClient.getSoleQueryResult("create table `test_automation." + bqSourceTable + "` as " +
+    BigQueryClient.getSoleQueryResult("create table `bq_automation." + bqSourceTable + "` as " +
```

Comment:
The value of dataset should come from plugin properties.

Reply:
Using a dataset constant instead of a hard-coded value.